### PR TITLE
Another attempt to fix stress tests

### LINKED ISSUE #82528: HttpStress execution failure on Windows
HttpStress tests are failing on Windows since 02.17:

```
The current Compose file version is not compatible with your engine version. Please upgrade your Compose file to a more recent version, or set a COMPOSE_API_VERSION in your environment.
```

https://dev.azure.com/dnceng-public/public/_build/results?buildId=180524&view=logs&j=0bc77094-9fcd-5c38-f6e4-27d2ae131589&t=4fd93899-d50b-5dc1-f9eb-66ab1b883c22

Weirdly, this happens after the actual run is complete.

### PR DESCRIPTION
**Pull request by antonfirsov**
Tried using another Windows image, which did not work (#82736), so decided to go with the workaround from https://github.com/actions/runner-images/issues/7080#issuecomment-1439684206 to fix #82528.

### CODE REVIEW

File: src/libraries/System.Net.Security/tests/StressTests/SslStress/run-docker-compose.ps1 [hunk 1]
```diff
@@ -15,6 +15,13 @@ Param(
 $REPO_ROOT_DIR = $(git -C "$PSScriptRoot" rev-parse --show-toplevel)
 $COMPOSE_FILE = "$PSScriptRoot/docker-compose.yml"
 
+# This is a workaround for an issue with 1es-windows-2022-open, which should be eventually removed.
+# See comments in <repo>/eng/pipelines/libraries/stress/ssl.yml for more info.
+$dockerComposeCmd = $env:DOCKER_COMPOSE_CMD
+if (!(Test-Path $dockerComposeCmd)) {
+    $dockerComposeCmd = "docker-compose"
+}
```
Comment (antonfirsov) [inline src/libraries/System.Net.Security/tests/StressTests/SslStress/run-docker-compose.ps1:23]: This is now an asymmetry between `run-docker-compose.ps1` in SslStress vs HttpStress.
For some reason, the HttpStress pipeline is ok to `docker-compose build` with V1 of compose but then run with V2, however in SslStress this resulted in a failure.
I don't want to add an equivalent workaround to the `.ps1` in HttpStress, when it's not necessary.

**Review by antonfirsov — COMMENTED** [review]
@dotnet/ncl this seems to work, can someone take a look?

**Review by rzikm — APPROVED** [review]
LGTM

**Pull request merged**